src/App.tsx

```diff
@@ -1,23 +1,30 @@
 import "./App.css";
 
 
-import { FbtParam, IntlViewerContext, fbt, init } from "fbt";
+import { FbtParam, FbtName, FbtEnum, IntlViewerContext, fbt, init, FbtPlural } from "fbt";
 import React, { FormEvent, useCallback, useRef, useState, ChangeEvent, ReactElement } from "react";
 import { Button, TextField, Container, Menu, MenuItem, Select, FormControl, makeStyles, Grid, colors } from '@material-ui/core';
 
 import intl from "./translatedFbts.json";
 import logo from "./logo.svg";
+import { stringify } from "querystring";
 
 // This will load the translated strings in FBT.
 init({ translations: intl });
 
-IntlViewerContext.locale = "fr_FR";
+const langStore = window.localStorage;
+if (langStore.getItem("lang") == null) {
+  langStore.setItem("lang", "en_US");
+}
+IntlViewerContext.locale = langStore.getItem("lang") || "en_EN";
 
-interface LanguageSelectorProps {
-  name: string;
+const supportedLanguages: { [shortname: string]: { name: string } } = {
+  "en_US": { name: fbt("English", "eng") },
+  "fr_FR": { name: fbt("French", "fr") },
+  "pi_PI": { name: fbt("Pirate", "pi") }
```

Comment:
The second string is supposed to be a description of what that string is supposed to be used for. This is to help translators to get more context about what they are translating.
Since we want this repository to be the best example possible, let's put something like "Label for language selector" to all of them.